app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_name, file_name, new_file_name = command.split()
```

Comment:
You need to validate the command. What will happens in command will be one of:
1. cp file1.txt file2.txt file3.txt
2. cp file1.txt